src/commands/Minion/mine.ts

```diff
@@ -49,6 +49,21 @@ const gloves = [
 	}
 ];
 
+const amulets = [
```

Comment:
just make this an array of strings with each name in it and might as well add all of the different versions for completeness

Reply:
the `resolveItems` method does this just use that